src/simplify_algebra.cpp

```diff
@@ -391,6 +392,23 @@ struct find_sub_const
     }
 };
 
+struct find_rsqrt
+{
+    auto matcher() const
+    {
+        return match::name("recip")(
+            match::args(match::name("sqrt")(match::args(match::any().bind("x")))));
```

Comment:
Should we consider output of sqrt used only once?

Reply:
fixed